bucket/firefox-esr.json

```diff
@@ -14,6 +20,7 @@
         }
     },
     "extract_dir": "core",
+    "post_install": "firefox-esr -CreateProfile \"Scoop-ESR $dir\\profile\"",
```

Comment:
What if this profile already exists? Shouldn't we check for it first?

Reply:
it will not affect `profiles.ini`, if profile `Scoop-ESR`(in this case) already exists.